app/uk/gov/hmrc/tradergoodsprofilesrouter/connectors/EISConnector.scala

```diff
@@ -104,6 +112,20 @@ class EISConnectorImpl @Inject() (
       .executeAndDeserialise[CreateRecordResponse]
   }
 
+  override def removeRecord(
+    eori: String,
+    recordId: String,
+    actorId: String,
+    correlationId: String
+  )(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Int] = {
+    val url = appConfig.eisConfig.removeRecordUrl
+    httpClientV2
+      .get(url"$url")(hc)
```

Comment:
is this suppose to be a get or put?

Reply:
it should be put, i will update this now, thanks